boards/xtensa/nxp_adsp_imx8ulp/Kconfig.defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NXP_ADSP_IMX8ULP
+
+config BOARD
+	default "nxp_adsp_imx8ulp"
+
+config DUMMY_DMA
```

Comment:
Why's this needed? We've no native dummy DMA driver ATM. On SOF side the dummy dma is compiled via zephyr/CMakeLists.txt based on the SoC.